<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubtitleRenamer</class>
 <widget class="QWidget" name="SubtitleRenamer">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Subtitle Renamer</string>
  </property>
  <widget class="QGroupBox" name="groupBox_formats">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string>Extensions</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_extensions">
     <item>
      <widget class="QLabel" name="label_video_format">
       <property name="text">
        <string>Video</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_video_format">
       <property name="text">
        <string>mkv</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_subs_format">
       <property name="text">
        <string>Subs Format</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_subs_format">
       <property name="text">
        <string>srt</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_video_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>181</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>Video List</string>
   </property>
   <widget class="QPlainTextEdit" name="plainTextEdit_video_list">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>361</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_subtitle_list">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>191</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>Subs List</string>
   </property>
   <widget class="QPlainTextEdit" name="plainTextEdit_subs_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>361</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit_folder">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>4</y>
     <width>343</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_folder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>30</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Folder</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_info">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>460</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_rename">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Rename</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_detect">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Detect</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
